raster/r.null/testsuite/test_r_null.py

```diff
@@ -0,0 +1,91 @@
+import grass.script as gs
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+from grass.gunittest.gmodules import SimpleModule
+
+
+class TestRNull(TestCase):
+    """Test case for r.null module"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up a temporary region and create test raster maps"""
+        cls.use_temp_region()
+        cls.runModule("g.region", n=3, s=0, e=3, w=0, res=1)
+
+        # Create map1: categories 1, 2, 3 as rows
+        cls.runModule(
+            "r.mapcalc",
+            expression="map_basic = if(row() == 1, 1, if(row() == 2, 2, if(row() == 3, 3, null())))",
```

Comment:
This could be simplified:
```
map_basic = row()
```

Reply:
@Shreshth-Malik It is generally a good idea to review your code for other occurrences of the same issue and fix those.